### ISSUE TITLE
Can't use `withRef`

### ISSUE THREAD

**Issue by mhahn** [erikras/redux-form-material-ui#4]
First, this is awesome.

I want to be able to focus a text field if someone hits an "edit" icon. I can't use `withRef` on the field though since the underlying component is a stateless functional component.

Happy to help translate these to `Component`

**Comment by mhahn** [comment]
Also, even when passing `withRef` correctly, it seems like the methods on the class have been dropped? I'm trying to get access to the `focus` method on `TextField` but it says its undefined:

![image](https://cloud.githubusercontent.com/assets/704593/15973610/49bd5818-2ef9-11e6-9c30-14776a298874.png)

vs an unwrapped field:

![image](https://cloud.githubusercontent.com/assets/704593/15973649/5f7e012a-2ef9-11e6-9376-1eab2f1d0a05.png)

**Comment by mhahn** [comment]
Another note is getting access to underlying methods on the material UI component (ie. `focus` on `TextField`).

I'm not sure if there is a cleaner way than doing something like:
```
class TextField extends Component {

  focus = () => {
    this.refs.field.focus()
  }

  render() {
    return <MaterialTextField ref='field' {...this.props} />
  }

}
```

**Comment by erikras** [comment]
Fixed in [`v2.2.0`](https://github.com/erikras/redux-form-material-ui/releases/tag/v2.2.0).

**Comment by mhahn** [comment]
@erikras you're my favorite react form developer ;)
